lib/services/registered_addons.pm

```diff
@@ -78,10 +79,21 @@ sub check_suseconnect {
     diag "@addons";
 }
 
+sub check_suseconnect_cmd {
+    my $ls_out     = script_output("SUSEConnect --list-extensions", 120);
+    my $status_out = script_output("SUSEConnect --status-text",     120);
```

Comment:
I think it is better save the output of the two cmds to log for later debug.

Reply:
No response for this comments yet.

Reply:
I have update code. add diag to log